starter-code/views/random.ejs

```diff
@@ -0,0 +1 @@
+<p><%= value %><p>
```

Comment:
This code es exactly the same that the one in joke-by-category.ejs. Remember DRY. Use only one file (e.g. show-joke.ejs) or do different things.